.ci.yaml

```diff
@@ -370,7 +369,6 @@ targets:
 
   - name: Mac clangd
```

Comment:
optional nit: Not critical, but this orchestrator could be a `Linux` host to save 5 minutes of mac host time.

Reply:
I mean I'm down - where can I read about that/how can I do it?

I am guessing orchestrator == `drone_dimensions`? Is there another example of this?

Reply:
So you'd change `Mac clangd` to something like `Linux mac_clangd`, and then add:
```
    drone_dimensions:
      - os=Linux
```
Like in `Linux mac_android_aot_engine`. Probably has to stay `bringup: true` in this PR to make sure that works properly.

Reply:
Done!